### ISSUE TITLE
After linking Github as an OAuth Provider I can no longer sign in with username/email and password. 

### ISSUE THREAD

**Issue by liltechnomancer** [pocketbase/pocketbase#5929]
If I create an account with username (or email) and password and then link my github account I no longer seem to be able to log in with username & password. I have tried resetting the password in the Pocketbase Admin UI thinking is may be related to this: https://github.com/pocketbase/pocketbase/security/advisories/GHSA-m93w-4fxv-r35v

But that did not resolve the issue. If I remove Github as an authorized provider, then I can log in with email & pass again.

I am mostly curious if this is intentional or a bug. If intentional I can just display a message when linking a github account. 

Thanks!

**Comment by ganigeorgiev** [comment]
I'm not able to reproduce it (testing with PocketBase v0.22.27). You can even test it with [presentator.io](https://presentator.io).

The linked issue tries to explain that if before the OAuth2 your account wasn't verified, then its password will be reset to a random one with the first OAuth2 sign-in. It has no relation to what auth methods are allowed.

Double check your collection configuration and what auth methods are allowed and make sure that there are no typos when entering your email/username + pass.

If you still think that it is a PocketBase issue, create a **minimal** reproducible repo OR at least provide some clear steps illustrating the problem with details about your collections (Admin UI > Settings > Export collections) and I'll try to have a look.

For other questions, please use the Q&A discussion.

**Comment by liltechnomancer** [comment]
Thanks for the fast reply. Today I was able to diagnose the problem and will detail it here incase others run into it. 

Upon trying to test user email resetting I learned that emails where not sending successfully. Maybe because I was running it in docker without SMTP set. 

This along with the following line in the security advisory:

> Additionally to warn existing users we now send an email alert in case the user has logged in with password but has at least one OAuth2 account linked. It looks something like:

Made me wonder if the email sending functionality needed to be in place for this auth to succeed. 

I added my SES SMTP credentials to pocketbase, which allowed test emails to succeed. Then upon re-testing I was able to login with Email/Password again. 

Thanks again!
